content/docs/1.7.0/references/settings.md

```diff
@@ -472,6 +473,12 @@ Number of seconds that Longhorn allows for the completion of replica rebuilding
 
 Setting that allows you to enable the V1 Data Engine.
 
+#### RWX Volume Fast Failover (Experimental)
+
+> Default: `false`
+
+Enable improved ReadWriteMany volume HA by shortening the time it takes to recover from a node failure.
```

Comment:
To avoid confusion, I think we can avoid using`HA` in the document.

```suggestion
Enable improved ReadWriteMany volume failover by shortening the time it takes to recover from a node failure.
```

cc @innobead